content/running_bookwyrm/external-storage.md

```diff
@@ -0,0 +1,120 @@
+Title: External Storage
+Date: 2021-06-07
+Order: 5
+
+By default, BookWyrm uses local storage for static assets (favicon, default avatar, etc.), and media (user avatars, book covers, etc.), but you can use an external storage service to serve these files. BookWyrm uses `django-storages` to handle external storage, such as S3-compatible services, Apache Libcloud or SFTP.
+
+## S3-compatible Services
+
+### Setup
+
+Create a bucket at your S3-compatible service of choice, along with an Acces Key ID and a Secret Access Key. These can be self hosted, like Ceph (LGPL 2.1/3.0) or MinIO (GNU AGPL v3.0), or commercial (Scaleway, Digital Ocean…).
```

Comment:
It took me a bit of looking to find the Digital Ocean instructions, so it might be handy to link them here: https://www.digitalocean.com/community/tutorials/how-to-create-a-digitalocean-space-and-api-key